libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -93,6 +93,7 @@ internal class MapboxTripSession(
                     .also { isUpdatingRoute = false }
             }
             is SetAlternativeRoutesInfo -> {
+                isUpdatingRoute = false
```

Comment:
I think this is not necessary, because `isUpdatingRoute` is set to `true` only for `BasicSetRoutesInfo`.

Reply:
Oh, you're right. It was a cherry-pick from 2.8 and it was not the case there.